sdk/appconfiguration/app-configuration/src/policies/throttlingRetryPolicy.ts

```diff
@@ -24,6 +25,57 @@ export function throttlingRetryPolicy(): RequestPolicyFactory {
   };
 }
 
+const StandardAbortMessage = "The operation was aborted.";
+
+/**
+ * A wrapper for setTimeout that resolves a promise after t milliseconds.
+ * @param delayInMs - The number of milliseconds to be delayed.
+ * @param abortSignal - The abortSignal associated with containing operation.
+ * @param abortErrorMsg - The abort error message associated with containing operation.
+ * @returns - Resolved promise
+ */
+export function delay(
```

Comment:
Can we put a comment on here so we know this is a duplicate of core-amqp's version.

Also, can we file an issue to unify these? This kind of delay seems useful enough that it'd be useful to have in a core library.

Reply:
Sounds good https://github.com/Azure/azure-sdk-for-js/issues/15796

Reply:
Will do the move when I update core-http as well.